plugins/flytekit-spark/flytekitplugins/spark/task.py

```diff
@@ -201,6 +195,21 @@ def pre_execute(self, user_params: ExecutionParameters) -> ExecutionParameters:
             sess_builder = sess_builder.config(conf=spark_conf)
 
         self.sess = sess_builder.getOrCreate()
+
+        if ctx.execution_state and ctx.execution_state.mode == ExecutionState.Mode.TASK_EXECUTION:
+            current_time = time.time()
+            current_dir = os.getcwd()
+            # Set the modification time of all files in the current directory to the current time
+            # since fast register doesn't preserve the modification time of the files and make_archive
```

Comment:
we arbitrarily set the time in fast register to 1970 just cuz.  it can be any constant.  if we make it 1980, would that make this cleaner?

Reply:
set it to 1980